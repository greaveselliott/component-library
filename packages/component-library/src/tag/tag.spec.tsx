import React from 'react';
import { shallow } from 'enzyme';

import Tag, { TagColors } from './tag.component';

const colors: TagColors[] = ['primary-300', 'shade-200'];

colors.forEach((color) =>
  test(`Renders ${color} tag`, () =>
    expect(shallow(<Tag color={color}>Abc</Tag>)).toMatchSnapshot()),
);
