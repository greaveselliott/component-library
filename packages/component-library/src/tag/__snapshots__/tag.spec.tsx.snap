// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders primary-300 tag 1`] = `
<div
  className="tag tag--color-primary-300"
>
  Abc
</div>
`;

exports[`Renders shade-200 tag 1`] = `
<div
  className="tag tag--color-shade-200"
>
  Abc
</div>
`;
